bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/EpsonProjectorDevice.java

```diff
@@ -210,6 +210,7 @@ public void disconnect() throws EpsonProjectorException {
         if (timeoutJob != null) {
             timeoutJob.cancel(true);
             this.timeoutJob = null;
+            ready = true;
```

Comment:
Good catch! You could move this above the if statement. Also the binding name [epsonprojector] should be in the title.

Reply:
thank you for updating the title. first pull request for openhab, so next time i can consider that. since approval already done, would you like my to update the line above the if or shall we leave it as it is? both ways are fine for me. could do the update this weekend if you like.